neo/app/actor/RequestRouter.scala

```diff
@@ -0,0 +1,72 @@
+package actor
+
+import akka.actor.{Actor, ActorLogging, ActorRef, Props}
+import akka.stream.Materializer
+import edu.uci.ics.cloudberry.zion.actor.BerryClient
+import edu.uci.ics.cloudberry.zion.actor.BerryClient._
+import edu.uci.ics.cloudberry.zion.common.Config
+import play.api.libs.json._
+import play.Logger
+
+import scala.concurrent.ExecutionContext
+
+class RequestRouter (out: ActorRef, berryClientProp: Props, config: Config)
+                    (implicit ec: ExecutionContext, implicit val materializer: Materializer) extends Actor with ActorLogging {
+
+  import RequestRouter._
+
+  val streamingBerryClient = context.actorOf(berryClientProp)
+  val nonStreamingBerryClient = context.actorOf(berryClientProp)
```

Comment:
we should either pass a function or directly create the `berry` inside.

pass a function 
```
class RequestRouter (out: ActorRef, berryClientProp: ActorRef => Props, config: Config)
          (implicit ec: ExecutionContext, implicit val materializer: Materializer) extends Actor with ActorLogging {
    val streamingBerryClient = context.actorOf(berryClientProp(out))
```